distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Zone.enso

```diff
@@ -140,4 +140,7 @@ type Time_Zone
 
     ## Compares two Zones for equality.
     == : Time_Zone -> Boolean
-    == self that = Time_Utils.equals_zone self that
+    == self that =
```

Comment:
I believe this can be written as:
```
== self that = case Meta.type_of that of
    Time_Zone -> Time_Utils.equals_zone self that
    _ -> False
```
that would mean less changes in the engine now. Moroever this kind of code would remain valid in the future even with [by type pattern matches](https://www.pivotaltracker.com/story/show/183188846)